### ISSUE TITLE
init settings shown in UI are misleading (do not match default)

### ISSUE THREAD

**Issue by vincentvialard** [OpherV/gitflow4idea#283]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] puppy => You're not submitting a puppy. I already have one and he's [adorable](https://twitter.com/Opherv/status/974341249099542528)

* **What is the current behavior?**

The setting for releases shown in the UI is not the default setting:
![gitflow current](https://user-images.githubusercontent.com/16098519/76774550-56655380-67a4-11ea-9c95-d1055ac17a5a.png)

Running the defaults uses "master" for production releases. 
Log:
~~~~
Which branch should be used for bringing forth production releases?
   - develop
   - master
Branch name for production releases: [master] 
Which branch should be used for integration of the "next release"?
   - develop
Branch name for "next release" development: [develop] 
How to name your supporting branch prefixes?
Feature branches? [feature/] 
Bugfix branches? [bugfix/] 
Release branches? [release/] 
Hotfix branches? [hotfix/] 
Support branches? [support/] 
Version tag prefix? [] 
~~~~

* **Is this a bug? Sorry about that. If so give me explicit details how to reproduce:**

1. Open IntelliJ
2. Create a new git project
3. Click the "no git workflow" message showing at the bottom right

* **What is the expected behavior?**

The UI should show the settings used per the default:
![gitflow expected](https://user-images.githubusercontent.com/16098519/76775025-0cc93880-67a5-11ea-9a07-59f814f5d0cc.png)

* **What is the motivation / use case for changing the behavior?**

It is confusing and I was reluctant to use the default settings...

* **Please tell me about your environment:**
  
  - Gitflow4idea version: 0.7.1
  - Gitflow version: 1.12.3 (AVH Edition)
  - 
~~~~
IntelliJ IDEA 2019.3.3 (Community Edition)
Build #IC-193.6494.35, built on February 11, 2020
Runtime version: 11.0.5+10-b520.38 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.11.6
GC: G1 Young Generation, G1 Old Generation
Memory: 1024M
Cores: 2
Registry: debugger.valueTooltipAutoShowOnSelection=true
Non-Bundled Plugins: Gitflow, VisualVMLauncher
~~~~

* **Other information** (e.g. detailed explanation, stacktrace, related issues, suggestions how to fix, links for me to have context words of praises, pictures of puppies (again with the puppy??) )

**Comment by vincentvialard** [comment]
Got another behaviour today (2 times "master" instead of "develop"):
![gitflow-default](https://user-images.githubusercontent.com/16098519/76859409-920e2500-6859-11ea-8c64-e58b8cc07778.png)

but same default settings were used, as shown in log:
~~~~
git -c credential.helper= -c core.quotepath=false -c log.showSignature=false flow init -d
Using default branch names.
Which branch should be used for bringing forth production releases?
   - master
Branch name for production releases: [master] 
Branch name for "next release" development: [develop] 
How to name your supporting branch prefixes?
Feature branches? [feature/] 
Bugfix branches? [bugfix/] 
Release branches? [release/] 
Hotfix branches? [hotfix/] 
Support branches? [support/] 
Version tag prefix? [] 
~~~~

**Comment by OpherV** [comment]
Good idea. I'll add this to next version

**Comment by OpherV** [comment]
Fix in 0.7.3 now out
